fw/fs.c

```diff
@@ -331,6 +353,10 @@ int fs_write(void* buf, uint16_t len) //{{{
 	if (fs_busy())
 		return FS_ERROR_WRITE_IN_PROGRESS;
 
+	// time of first write will be stored into file startblock when closed
```

Comment:
change this to say:

// the rtc time stored in the file is the time of the first write